defaults/main.yml

```diff
@@ -1,3 +1,10 @@
 ---
 scylladb_repo_url: ""
 scylladb_version: ""
+scylladb_cluster_name: ""
+scylladb_seed_servers: ""
+scylladb_developer_mode: "true" # This should be set to false for production
```

Comment:
Documentation to README.md for this.

Reply:
Removed seed server var and provided info on cluster_name and developer_mode